nservicebus/upgrades/asp-saga-deduplication.md

```diff
@@ -10,69 +10,67 @@ related:
+6. Update NServiceBus.Azure dependency to version 6.2.4 or higher in all endpoints that use it and release the updated endpoints.
+7. Run de-duplication utility again (see below for details). This will fix problem saga data or list conflicts that were introduced to the data store while steps #5 and #6 were being performed.
+8. If the de-duplication utility has output any classes that require the identification of a correlation property, then return to step #5 above and address those classes.
+
+### Running the de-duplication utility
+1. Open a commandline and run the following command: `NServiceBus.AzureStoragePersistence.SagaDeduplicator.exe directory=<directory> operation=Download` where:
+	- **directory**: This should be the full path of the working directory that the de-duplication tool will use. The tool will need to have read, write and delete permissions on the folder selected. If there are any spaces in the directory path this value should be enclosed in double quotes. If the tool was run previously, ensure that the directory indicated is empty. Example: `directory="C:\my working folder\deduplication"`
+2. The utility will output a list of saga data classes that need to have a unique correlation property configured on them. Make note of this list for use in later steps.
+3. The de-duplication utility will have fixed the data for all saga instances that it could. There can, however, be situations where the utility was unable to resolve the correct saga instance. When this happens the de-duplication tool will download conflicting sagas to the `directory\sagaTypeName`. Every set of conflicting sagas is downloaded to a separate directory containing files named with conflicting sagas identifiers. For example, running the utility with `directory=data` against an assembly that has a `TwoInstanceSagaState` class that implements `IContainSagaData` will result in a following structure:
+```
+data
+└───TwoInstanceSagaState
+    └───d10c2f15-06d2-1370-e0ee-781710b5d598
+        └───0e36dc9a-eec0-455c-a4d3-b8b275711d15
+        └───8ee2f4b2-eaf2-4d12-a87e-a5e000aaa815
 ```
-4. what about conflicts?
-5. Run the following command: `NServiceBus.AzureStoragePersistence.SagaDeduplicator.exe directory=<directory> sagaTypeName=<sagaTypeName> sagaProperty=<sagaProperty> operation=Upload connectionString=<connectionString>`
-	- All of the commandline parameters, with the exception of `operation=Upload`, should be exactly the same as they were in step #3.
+where `0e36dc9a-eec0-455c-a4d3-b8b275711d15` and `8ee2f4b2-eaf2-4d12-a87e-a5e000aaa815` contain JSON payload of conflicting sagas. Next to the saga properties there's an additional property to mark the saga as selected. Initially, its value is `"$Choose_this_saga": false`. For each set of conflicting sagas, one of them should be updated accordingly and marked as the chosen with `"$Choose_this_saga": true`.
```

Comment:
The example payloads are not included. Do you think that they're uneeded? See: https://github.com/Particular/docs.particular.net/pull/1387#discussion_r58683222